README.md

```diff
@@ -30,7 +30,7 @@ export default () => {
   const app = new App(<div></div>);
 
   const Session = app.plugin(JWTSession, {secret: __NODE__ && 'secret here'});
-  const {fetch} = app.plugin(CsrfProtection, {Session});
+  const {fetch, ignore} = app.plugin(CsrfProtection, {Session}).of();
```

Comment:
Would whitelist be more clear?

Reply:
I think it should be default on, so that explicit opting out means a conscious decision to be unsafe